src/utils/checkONSASFields.m

```diff
@@ -22,10 +22,27 @@ function checkONSASFields( materials, elements, boundaryConds, initialConds, mes
   checkFields(elements, {'elemType', 'elemTypeParams','massMatType',...
                             'elemCrossSecParams','aeroNumericalParams','aeroCoefFunctions','chordVector'});
 
-%   checkFields(elements, {'elemType', 'elemTypeParams','massMatType',...
-%                             'elemCrossSecParams','aeroNumericalParams','aeroCoefFunctions','chordVector'});
-%                             checkFields(elements, {'elemType', 'elemTypeParams','massMatType',...
-%                             'elemCrossSecParams','aeroNumericalParams','aeroCoefFunctions','chordVector'});
+   analysisSettingsDefaultFields = {'geometricNonLinearAero', 'numGaussPointsAeroForce','computeAeroStiffnessMatrix',...
```

Comment:
In the future, I recommend reading this directly from `setDefaults.m`. Something like `defaultONSASStructs()` and then fetch all the field names from that. Probably: https://www.mathworks.com/help/matlab/matlab_oop/class-constructor-methods.html is what we want in the long term.

Reply:
indeed. oop in ONSAS is in the horizon for the looong term. I'd not do these changes in the short term.

Reply:
opening an issue for this would be a good idea. i am resolving this conversation for this PR

Reply:
Done